### DISCUSSION TITLE
In an example, there are two rectangles. How to change the color of one of them

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#7780] [Q&A]
![444](https://user-images.githubusercontent.com/89827437/132233090-f3039b62-8ac1-4141-b65c-530c2805a2f1.png)

**Comment by domis86** [comment]
in both calls to `graphic.beginFill` you passed same color `0xff0000` - try pass different color in one of them.

Also you you may want to have 2 `PIXI.Graphics` object as described in https://github.com/pixijs/pixijs/discussions/7779#discussioncomment-1286725
